HostDependencies.md

```diff
@@ -150,15 +141,15 @@ sudo apt install haskell-stack
 Install the following packages on your Ubuntu machine:
 
 ```bash
-sudo apt install -y \
+sudo apt install \
     clang gdb libssl-dev libclang-dev \
     libcunit1-dev libsqlite3-dev qemu-kvm
 ```
 #### Fedora 
-> *On Fedora 30 or later*
+> *Not working yet*
```

Comment:
Does this mean that these instructions don't actually work?
If they do not, please remove them from this PR.
If they do work, remove this line.

Reply:
This is deleted.